src/services/initializer-service.ts

```diff
@@ -112,15 +114,25 @@ export class InitializerService {
 
     this.createDefaultDirectories()
 
-    const bundleDescriptorService = new BundleDescriptorService(
-      this.filesys.getBundleDirectory()
-    )
-    const descriptor = bundleDescriptorService.getBundleDescriptor()
-    bundleDescriptorService.writeBundleDescriptor({
-      ...descriptor,
-      name,
-      version
-    })
+    try {
+      const bundleDescriptorService = new BundleDescriptorService(
+        this.filesys.getBundleDirectory()
+      )
+      const descriptor = bundleDescriptorService.getBundleDescriptor()
+      bundleDescriptorService.writeBundleDescriptor({
+        ...descriptor,
+        name,
+        version
+      })
+    } catch {
+      if (process.env.ENTANDO_BUNDLE_CLI_NO_JSON_WARNING === 'false') {
```

Comment:
it looks a bit too generic catch. let's explicitly check for entando.json existence

Reply:
done. I added logic on checking the error.message to ensure that it is catching with the right error